dagql/idtui/frontend.go

```diff
@@ -24,6 +25,13 @@ import (
 	sdklog "go.opentelemetry.io/otel/sdk/log"
 )
 
+// having a bit of fun with these. cc @vito @jedevc
+var skipLoggedOutTraceMsgEnvs = []string{"NOTHANKS", "SHUTUP", "GOAWAY", "STOPIT"}
```

Comment:
Love that you actually did this 😁❤️

Just making sure that we honor all of them all the time right? Otherwise our joke won't be nearly as funny...

Reply:
yep, we do